docs/tutorials/52-observe-application-metrics.md

```diff
@@ -15,12 +15,9 @@ This tutorial focuses on the `cpu_temperature_celsius` metric, that is one of th
 
 ## Prerequisites
 
-To complete the tutorial, you must meet one of these prerequisites and have:
-
-- A cluster with Kyma 1.3 or higher
-- Kyma 1.3 or higher installed locally with the Monitoring module
-
-> **NOTE:** The monitoring module is not installed by default as a part of the [Kyma Lite](/root/kyma/#installation-overview) package.
+- You have a cluster with Kyma 2.0 or higher.
+- You have installed Kyma 2.0 or higher locally with the Monitoring module.
```

Comment:
Do you know what `locally` means in this context? Cannot we just merge those 2 bullet points?

Reply:
Excellent question.  I'm not exactly sure why the bullet points are distinct.
They came from the current [Monitoring prerequisites](https://kyma-project.io/docs/components/monitoring/#tutorials-observe-application-metrics-prerequisites), I just updated the Kyma version number as I figured we're talking about K2.0.

Reply:
(removed entirely)